### PR TITLE
Modify update strategies to reference labels

### PR DESCRIPTION
**Pull request by pipex**
Change-type: patch

---

> *Please make sure to read the [CONTRIBUTING](https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md) document before opening the PR for relevant information on contributing to the documentation. Thanks!*

### CODE REVIEW

File: pages/learn/deploy/release-strategy/update-strategies.md [hunk 1]
```diff
@@ -8,19 +8,22 @@ excerpt: Choosing an update strategy for your {{ $names.company.lower }} devices
 With the {{ $names.company.lower }} device supervisor version 1.3, we added the ability to choose the update strategy on devices, that is, the order and way in which the steps to perform an update are executed. You can check whether your Supervisor has the appropriate version in the "Supervisor version" entry in the device dashboard page.
 These update strategies allow users to choose between four modes that are suited for different applications, depending on available resources and the possible need to have a container running at all times.
 
-Update strategies can be applied in the [Configuration][configuration] section. The two variables that are involved are:
+Update strategies can be applied by setting a [docker-compose label](labels). Two labels that are involed are:
```
Comment (klutchell) [inline pages/learn/deploy/release-strategy/update-strategies.md:11]:
```suggestion
Update strategies can be applied by setting a [docker-compose label](labels). Two labels that can be invoked are:
```

File: pages/learn/deploy/release-strategy/update-strategies.md [hunk 2]
```diff
@@ -8,19 +8,22 @@ excerpt: Choosing an update strategy for your {{ $names.company.lower }} devices
...
 * [`download-then-kill`](#download-then-kill),
 * [`kill-then-download`](#kill-then-download),
 * [`delete-then-download`](#delete-then-download), and
 * [`hand-over`](#hand-over),
 
-which are explained below. `BALENA_SUPERVISOR_HANDOVER_TIMEOUT` is only used in the  `hand-over` strategy, and its use is explained in the [strategy's description](#hand-over).
+which are explained below. The label `io.balena.update.handover-timeout` is only used in the  `hand-over` strategy, and its use is explained in the [strategy's description](#hand-over).
+
+__Note:__ Prior to Balena Supervisor v7.23.0 the strategy was controlled by two [configuration](configuration) variables, `RESIN_SUPERVISOR_UPDATE_STRATEGY` and `RESIN_SUPERVISOR_HANDOVER_TIMEOUT`, which had the same effect as the labels do today. 
+Such mechanism for controlling the strategy is considered _deprecated_ and may be removed in the future.
```
Comment (klutchell) [inline pages/learn/deploy/release-strategy/update-strategies.md:26]:
```suggestion
This mechanism for controlling the strategy is considered _deprecated_ and may be removed in the future.
```

File: pages/learn/deploy/release-strategy/update-strategies.md [hunk 3]
```diff
@@ -65,10 +68,11 @@ Its behavior is as follows:
 * The old and new releases should communicate between each other so that the old one frees any resources that the new one needs (e.g. device files, database locks, etc) and the new version can start running fully.
 * Once this "handover" is performed between the releases (old or new), your service must signal to the Supervisor that the old version is ready to be killed, by creating a file at `/data/resin-kill-me`.
 * When the Supervisor detects that the file has been created, the Supervisor kills the old container and deletes it from disk.
-* If the file is not created after a time defined in `BALENA_SUPERVISOR_HANDOVER_TIMEOUT`, the Supervisor kills the old version.
+* If the file is not created after a time defined by the label `io.balena.update.handover-timeout`, the Supervisor kills the old version.
```
Comment (klutchell) [inline pages/learn/deploy/release-strategy/update-strategies.md:71]:
```suggestion
* If the file is not created after a time defined by the `io.balena.update.handover-timeout` label, the Supervisor kills the old version.
```

File: pages/learn/deploy/release-strategy/update-strategies.md [hunk 4]
```diff
@@ -8,19 +8,22 @@ excerpt: Choosing an update strategy for your {{ $names.company.lower }} devices
...
-Setting `BALENA_SUPERVISOR_UPDATE_STRATEGY` to a valid value selects the update strategy. The possible values are:
+Setting `io.balena.update.strategy` to a valid value selects the update strategy. The possible values are:
 
 * [`download-then-kill`](#download-then-kill),
 * [`kill-then-download`](#kill-then-download),
 * [`delete-then-download`](#delete-then-download), and
 * [`hand-over`](#hand-over),
 
-which are explained below. `BALENA_SUPERVISOR_HANDOVER_TIMEOUT` is only used in the  `hand-over` strategy, and its use is explained in the [strategy's description](#hand-over).
+which are explained below. The label `io.balena.update.handover-timeout` is only used in the  `hand-over` strategy, and its use is explained in the [strategy's description](#hand-over).
```
Comment (klutchell) [inline pages/learn/deploy/release-strategy/update-strategies.md:23]:
```suggestion
which are explained below. The `io.balena.update.handover-timeout` label is only used in the  `hand-over` strategy, and its use is explained in the [strategy's description](#hand-over).
```

File: pages/learn/deploy/release-strategy/update-strategies.md (hunk 1 above)
Comment (pipex) [reply pages/learn/deploy/release-strategy/update-strategies.md:11]: I think I meant "involved" here :p, nice catch @klutchell. Do you think "invoked" works better? Can you invoke a label?

Comment (pipex) [reply pages/learn/deploy/release-strategy/update-strategies.md]: I think "involved" makes more sense in the full context of the phrase. "Update strategies can be applied by setting docker-compose label. The two labels that are involved are"

**Pull request merged**